helm/cluster-azure/values.schema.json

```diff
@@ -333,6 +333,15 @@
                         "title": "Disable HealthChecks for the MachineDeployment",
                         "type": "boolean"
                     },
+                    "failureDomain": {
+                        "enum": [
+                            "1",
+                            "2",
+                            "3"
```

Comment:
Is every Azure region consist of three failureDomains? Or can there be Azure regions with only one of them?